apps/worker/src/app/workflow/usecases/handle-last-failed-job/handle-last-failed-job.usecase.ts

```diff
@@ -20,7 +18,7 @@ export class HandleLastFailedJob {
     private createExecutionDetails: CreateExecutionDetails,
     private queueNextJob: QueueNextJob,
     private jobRepository: JobRepository,
-    private workflowRunRepository: WorkflowRunRepository,
+    // private workflowRunService: WorkflowRunService,
```

Comment:
Why is this hae comments? I guess a mistake?

Reply:
this is a leftover of redundant code, i will remove it, the run job use case is responsible for updating.